bot.py

```diff
@@ -1341,7 +1341,17 @@ async def auto_link(ctx):
 
     await ctx.respond(content=message_content, ephemeral=True)
 
-
+@bot.translate_command(
+    description = "Automatically translate any language into another using Google Translate API", aliases = ['tr']
+)
+async def translate(ctx,lang_to, *args):
+    lang_to = lang_to.lower()
+    if lang_to not in googletrans.LANGUAGES and lang_to not in googletrans.LANGCODES:
+        raise bot.BadArgument("Unfortunately the bot cannot translate this. Please contact mods and devs for more guidance.")
+    text = ' '.join(args)
+    translator = googletrans.Translator()
+    text_translator = translator.translate(text, dest=lang_to).text
+    ctx.send(text.translated)
```

Comment:
Does the `text` object, which seems to be a default Python string, have any attribute `.translated`? Is this supposed to be one of the objects built by `googletrans` lib?